.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: gomod
```

Comment:
might want to group golang.org/x together so it's a single PR instead of 3 when these get bumped